metaseq/criterions/cross_entropy.py

```diff
@@ -187,7 +187,7 @@ def serialize_tensor_to_numpy(t):
                             log_line = {
                                 "id": int(logging_output["id"][i]),
                                 "t":  serialize_tensor(logging_output["targets"][i]),
-                                "p":  serialize_tensor(logging_output["log_probs"][i]),
+                                "p":  serialize_tensor(logging_output["log_probs"][i], multiplier=10),
```

Comment:
@ktirumalafb note to self, the multiplier is set back to 10 here (keep this for now)